package.json

```diff
@@ -98,9 +98,6 @@
   },
   "lint-staged": {
     "*.ts": [
-      "eslint --fix",
```

Comment:
q: is this intentionally removed or what? if so, why remove it?

Reply:
intentionally. addded back!